packages/react-instantsearch/src/templates/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Carousel';
```

Comment:
it's worth thinking about, but in vanilla it's called /templates because that exists as a concept already, in react the components end in Component, so they should be in a components folder I think (or widgets is probably fine too, Highlight isn't a widget per se either and is exported from widgets)

Reply:
This or `layout`. It doesn't change anything for users who will import from `react-instantsearch`, but internally it makes more sense.